### PR TITLE
add internet gateway, fix missing variables, add warning about ami

### LINKED ISSUE #1: No internet gateway
Heyo,

I came across this project, and while it works about 90% of the way, the appliance never becomes accessible because no internet gateway is created.  I'll offer up a PR that I managed to get working.

### PR DESCRIPTION
**Pull request by StephanX**
Without the internet gateway component, the new instance can't communicate with the internet.  Also, fix variables "private_key", "ssh_user", and "ssh_port"

Closes #1

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -8,20 +8,50 @@ variable "subnet_cidr_block" {
 
 resource "aws_vpc" "main" {
   cidr_block = "${var.vpc_cidr_block}"
+  tags {
+    enable_dns_support = true
+    enable_dns_hostnames = true
```
Comment (lmammino) [inline main.tf]: I believe these 2 lines should be outside the `tags` block

File: main.tf [hunk 2]
```diff
@@ -8,20 +8,50 @@ variable "subnet_cidr_block" {
...
   vpc_id     = "${aws_vpc.main.id}"
+  map_public_ip_on_launch = true
   cidr_block = "${var.subnet_cidr_block}"
 }
 
+resource "aws_internet_gateway" "gw" {
+  vpc_id = "${aws_vpc.main.id}"
+  tags {
+        Name = "Internet Gateway for openvpn"
+    }
```
Comment (lmammino) [inline main.tf]: code doesn't look formatted here

File: main.tf [hunk 3]
```diff
@@ -8,20 +8,50 @@ variable "subnet_cidr_block" {
...
+}
+
+resource "aws_eip" "openvpn_eip" {
+  vpc      = true
+  depends_on = ["aws_internet_gateway.gw"]
+}
```
Comment (lmammino) [inline main.tf]: Code is not formatted here

**Review by lmammino — CHANGES_REQUESTED** [review]
Great work, thanks a million for contributing :)

I left you a couple of minor things I believe needs to be addressed before merging, if you can take care of them

**Comment by StephanX** [comment]
Commit amended, ref: https://github.com/lmammino/terraform-openvpn/pull/2/commits/4ccef93f59d80aaefb15016bdc0331a79209b32c

**Comment by lmammino** [comment]
Added some formatting, and now ready to be merged! Thanks a million 👍

**Review by lmammino — APPROVED** [review]

**Comment by StephanX** [comment]
Clean code is good code!  Glad I could help.

**Pull request merged**